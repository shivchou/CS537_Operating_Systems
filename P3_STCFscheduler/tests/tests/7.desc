Testing remain() system call with invalid values
