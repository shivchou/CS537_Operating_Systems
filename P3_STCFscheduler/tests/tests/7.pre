./edit-makefile.sh ../solution/Makefile test_7 > ../solution/Makefile.test
cp -f tests/test_7.c ../solution/test_7.c
cd ../solution/
make clean
make -f Makefile.test clean
make -f Makefile.test xv6.img
make -f Makefile.test fs.img
cd ../tests
