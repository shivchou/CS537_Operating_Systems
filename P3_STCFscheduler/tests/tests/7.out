XV6_TEST: scheduler : STCF
XV6_TEST: remain() test started
XV6_TEST: remain(0) returned -1
XV6_TEST: remain(-50) returned -1
XV6_TEST: pid 4 started
XV6_TEST: pid 4 finished
XV6_TEST: remain() test finished
